docs/index.md

```diff
@@ -42,3 +42,14 @@ dependencies {
 
 Artifacts of SDA Spring Boot Commons are available [at Maven Central](https://search.maven.org/search?q=g:org.sdase.commons.spring.boot)
 since release [0.11.2](https://github.com/SDA-SE/sda-spring-boot-commons/releases/tag/0.11.2).
+
+### Static directories
+Since Spring Boot runs in an embedded Tomcat server, it needs some tmp directories to support the container run in a readonly file system.
+Therefore, your application need to set a folder called `static` and a folder structure `tmp/tomcat` in the root directory of your container.
+In order to do so, create a folder `static` and `tmp/tomcat` on `src/main/jib`.
+When the container's image is generated with `gradlew jibDockerBuild`, these folders will be copied to the container.
+In case you need the tmp folder to be writable, you can mount a volume in your container and set the environment variable pointing to your folder:
+
+```
+SERVER_TOMCAT_BASEDIR=/tmp/tomcat
+```
```

Comment:
Isn't this covered by the [predefined property](https://github.com/SDA-SE/sda-spring-boot-commons/pull/151/files#diff-948deeb2743800b0019af9f1f614f42abc071037669b91c7b23840b27eddc303R5)?

If so:
```suggestion
In case you need the tmp folder to be writable, you can mount a writable volume to `/tmp` in your container.
```

Reply:
I will adjust to say that they can change to another location, if they prefer.